.buildkite/pipeline.yml

```diff
@@ -31,6 +31,20 @@ steps:
       - github_commit_status:
           context: Prototype Build
 
+  - group: ":rocket: Prototype Build"
+    if: build.branch == trunk
+    steps:
+      - input: Deploy Prototype Build
+        prompt: Share a Prototype Build via Firebase App Distribution?
+        key: prototype_triggered
+      - label: Prototype Build
+        depends: prototype_triggered
+        command: .buildkite/commands/prototype-build.sh
```

Comment:
I considered splitting build from upload as part of this PR, but decided to save it log it as a nice to have for a future date instead.

Doing so would have required updating the existing prototype build step and I didn't want to grow the scope.